### PR TITLE
feat: added breakout box config and styles

### PR DESCRIPTION
**Pull request by rkelbel**
## Summary
This pull request gives content editors the ability to add "Breakout Box" paragraph styles to pages. 

This pull request adds the following:
- Config for CKEditor/WYSIWYG types.
- Styling for Humsci Basic themes.
- New class: (`hs-well`) and styling for Humsci Legacy theme.

I opted to use the class ('hs-well') instead of ('hb-well') to keep this consistent with other inline paragraph syles within WYSIWYG/CKEditor. They use the same code for styling, but it was updated to a mixin since it has a more universal use. 
Info detailed in [this Jira card.](https://sparkbox.atlassian.net/browse/STN-365)

## Browsers Tested
The following browser testing has been completed:
- [x] Chrome
- [x]  Firefox
- [x]  Safari
- [x]  Edge
- [x]  IE 11

## Steps to Test
1. Run npm test
2. Import changes to the ckeditor config files.
3. Create a basic page, (content > basic page) and add Breakout Box text to the page within the WYSIWYG editor style options.
4. Confirm the styling for Breakout Box https://sparkbox-sandbox-prod.stanford.edu/qa/breakout-and-quote-styles shows up in the WYSIWYG while content editing and the option in text styles exists.
image
5. Confirm the styling shows up correctly on the page view after saving the page.
image
6. Add a variety of style options as outlined in the Jira card, within a row, within an accordion, etc.
7. Test that same page in the different appearance themes: Colorful, Traditional and the Legacy theme.

(I was able to test images floating next to breakout boxes in rows, but they did not allow placement within the breakout box. If this functionality needs changed, I will want to pair with someone to discuss this ability.)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

**Review by kaseybon — APPROVED** [review]

File: docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss [hunk 1]
```diff
@@ -181,3 +181,17 @@
     background-image: linear-gradient(to $icon-position, var(--palette--tertiary-highlight) $icon-width, transparent $icon-width);
   }
 }
+
+@mixin hb-well {
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss:185]: 👏

File: docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss [hunk 2]
```diff
@@ -181,3 +181,17 @@
...
+    @include hb-global-color('background-color', 'gray-light');
+  }
+
+  @include hb-traditional {
+    @include hb-pairing-color('background-color', 'tertiary-highlight');
+  }
+
+  padding: hb-calculate-rems(20px) hb-calculate-rems(18px);
+  border: 0 none;
+  display: table;
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss:196]: what is display: table for?

Comment (silverli) [reply docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss:196]: @kaseybon maybe you know?

Comment (kaseybon) [reply docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss:196]: This is what makes the breakout box work with an image is floated left or right nearby. Without it the background bleeds behind the image and looks broken.

**Review by silverli — COMMENTED** [review]
@kaseybon according to what I've seen, the following CSS changes happened:

well.scss was added, which altered the compiled css for css/base/index.css

media.scss was changed, which altered the compiled css for css/ckeditor.css

media.scss was changed, which also altered css/components/atoms.index.css

**Review by silverli — APPROVED** [review]
I approve!

**Pull request merged**